sw/device/boot_rom2/boot_rom.c

```diff
@@ -5,8 +5,10 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "sw/device/boot_rom2/final_jump.h"
 #include "sw/device/boot_rom2/uart_log.h"
 #include "sw/device/lib/base/log.h"
+#include "sw/device/lib/base/stdasm.h"
```

Comment:
Is this used in this file?

Reply:
It was, and I forgot to delete this line. Thanks!